server/src/train/train.cpp

```diff
@@ -186,6 +188,14 @@ Train::Train(World& world, std::string_view _id) :
   Attributes::addObjectEditor(throttleSpeed, false);
   m_interfaceItems.add(throttleSpeed);
 
+  Attributes::addMinMax(accelerationRate, 0.01, 10.0);
+  Attributes::addUnit(accelerationRate, "m/s^2");
+  m_interfaceItems.add(accelerationRate);
+
+  Attributes::addMinMax(brakingRate, -10.0, 0.01);
```

Comment:
max must be `-0.01` I think.

Reply:
Done this as welllllll.
see it's complicated working on negatives

Reply:
yeah, all positive like you suggested is much better :)